plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/CachingHiveMetastoreConfig.java

```diff
@@ -25,6 +25,11 @@
 
 public class CachingHiveMetastoreConfig
 {
+    public static boolean isCachingHiveMetastoreConfigRelevantForCaching(Duration metastoreCacheTtl, long metastoreCacheMaximumSize)
```

Comment:
can it `isCacheEnabled` and put it below non-static `isCacheEnabled`

Reply:
Applied